lib/model.php

```diff
@@ -76,10 +80,18 @@ public function purge()
      */
     private function _getStore()
     {
+        // FIXME
+        // Workaround so that config value don't need to be changed
```

Comment:
Hmm, yeah. If we want to be 100% compatible I'd say we need this workaround.
But we could also just persuade server admins to change the config...